utils/Constants.ts

```diff
@@ -1,60 +1,108 @@
-  mainnetFork: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
-};
-
-// we only ue this feed in tests moce it out
-export const BTC_USD_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c"
-};
-
-export const BTC_ETH_FEED_ADDRESS: { [key: string]: string } = {
-  mainnetFork: "0xdeb288F737066589598e9214E782fa5A8eD689e8"
+// TODO: Do we really need it when Blindex doesn't host the DEX?
+export const EXTERNAL_USD_STABLE: { [key: string]: { symbol: string; address: string; decimals: number } } = {
+  mainnetFork: { symbol: "USDC", address: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48", decimals: 6 },
+  rsk: { symbol: "XUSD", address: "0xb5999795BE0EbB5bAb23144AA5FD6A02D080299F", decimals: 18 }
 };
 
+// TODO: Do we really need it?
 export const NATIVE_TOKEN_NAME: { [key: string]: string } = {
   mainnetFork: "ETH",
-  rsk: "RBTC"
+  rsk: "RBTC",
+  arbitrumTestnet: "ETH"
 };
 
+// TODO: Do we really need it?
 export const SECONDARY_COLLATERAL_TOKEN_NAME: { [key: string]: string } = {
   mainnetFork: "BTC",
-  rsk: "ETHs"
+  rsk: "ETHs",
+  arbitrumTestnet: "WBTC"
+};
+
+// TODO: Maybe combine it with the price feeds we deploy? That way we could have a field called "shouldUpdate" and the BE bot could read it to know if it should update it or not. That way we won't need it hardcoded on the BE config like it is right now (in the updaterSettings section in the BE environment file).
+export const chainlinkPriceFeeds: { [key: string]: ChainlinkPriceFeed } = {
+  EUR_USD_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0xb49f677943BC038e9857d61E7d053CaA2C1734C1"
+    },
+    arbitrumTestnet: null,
+    rsk: null
+  },
+  XAU_USD_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0x214ed9da11d2fbe465a6fc601a91e62ebec1a0d6"
+    },
+    arbitrumTestnet: null,
+    rsk: null
+  },
+  GBP_USD_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0x5c0ab2d9b5a7ed9f470386e82bb36a3613cdd4b5"
+    },
+    arbitrumTestnet: null,
+    rsk: null
+  },
+  ETH_USD_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419"
+    },
+    arbitrumTestnet: {
+      address: "0x5f0423B1a6935dc5596e7A24d98532b67A0AeFd8"
+    },
+    rsk: null
+  },
+  BTC_USD_FEED_ADDRESS: {
+    // This is not being used today as it was only needed on RSK but we don't yet have Chainlink there.
+    // Keeping it here for future use when Chainlink will deploy on RSK.
+    mainnetFork: {
+      address: "0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c"
+    },
+    arbitrumTestnet: {
+      address: "0x0c9973e7a27d00e656B9f153348dA46CaD70d03d"
+    },
+    rsk: null
+  },
+  BTC_ETH_FEED_ADDRESS: {
+    mainnetFork: {
+      address: "0xdeb288F737066589598e9214E782fa5A8eD689e8"
+    },
+    arbitrumTestnet: {
+      address: "0x6eFd3CCf5c673bd5A7Ea91b414d0307a5bAb9cC1"
+    },
+    rsk: null
+  }
+};
+
+export const importantAddresses: ImportantComponentsAddresses = {
```

Comment:
let's try to find another name for this object, "important" is not clear - what is considered to be important?
if we don't find a good name maybe those addresses shouldn't be defined under one object and better them separately

Reply:
Any suggestions?
I tried finding something like InfastructureAddresses

The reason I want to group them together is that otherwise this file becomes huge with used constants (like you see in the PR I deleted a lot)

Reply:
Another reason is that you need them per chain, so also didn't want to start doing it to each of them individually

Reply:
so maybe a better way will be to store cross-chain constants in this file and create a dedicated constant file for each chain... this way the separation will be between cross-chain and chain-specific configs.

we can also consider creating an interface for those configs if there are many common properties

Reply:
Check it out now, let me know what you think